File: 1 - BEGINNER/C/1020 - Idade em Dias.c
#include <stdio.h>
 
int main() {
 
    int dias_idade, resto, ano, mes, dia;
    
    scanf("%d", &dias_idade);
    
    ano = dias_idade / 365;
    resto = dias_idade % 365;
    
    mes = resto / 30;
    resto = resto % 30;
    
    dia = resto;
    
    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", dia);
    
    return 0;
}

File: 1 - BEGINNER/Java/1020 - Idade em Dias.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        int dias = input.nextInt();
        int anos = dias / 365;
        dias = dias % 365;
        int meses = dias / 30;
        dias = dias % 30;
        
        System.out.printf("%d ano(s)\n", anos);
        System.out.printf("%d mes(es)\n", meses);
        System.out.printf("%d dia(s)\n", dias);
        
        input.close();
 
    }
 
}